spring-cloud-data-rest/src/main/java/org/springframework/cloud/data/rest/config/AdminConfiguration.java

```diff
@@ -87,27 +86,9 @@ public ModuleDeployer moduleDeployer(ModuleLauncher moduleLauncher) {
 
 	@Configuration
 	@Profile("cloud")
+	@Import(CloudConfiguration.class)
```

Comment:
would this work if the `@Import` were at class level in AdminApplication, and then `@Profile` were at class level for CloudConfiguration?